.github/stale_example.yml

```diff
@@ -0,0 +1,69 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+ daysUntilStale: 76
```

Comment:
Fixed according to feedback, 90 days 3 moth in total
daysUntilStale: 76
daysUntilClose: 14